### PR TITLE
Added missing punctuation & code block

### PR DESCRIPTION
**Pull request by sumitkharche**

### CODE REVIEW

File: deployment/heroku.md [hunk 1]
```diff
@@ -97,23 +97,23 @@ Go to the `api/` directory, then
 
 1. Initialize a git repository:
 
-    git init
+    `git init`
```
Comment (dunglas) [inline deployment/heroku.md]: All these changes must be reverted. It's valid Markdown, rendered as a code block.

Comment (sumitkharche) [reply deployment/heroku.md]: @dunglas I have reverted the changes.

Comment (sumitkharche) [reply deployment/heroku.md]: @dunglas Did you got chance to look into this PR.

File: deployment/kubernetes.md [hunk 2]
```diff
@@ -95,4 +95,4 @@ We noticed that some tiller RBAC trouble occurred. You can usually resolve it by
       deployment "tiller-deploy" patched
 
 Please, see the [related issue](https://github.com/kubernetes/helm/issues/3130) for further details / information.
-You can also take a look at the [related documentation](https://github.com/kubernetes/helm/blob/master/docs/rbac.md)
+You can also take a look at the [related documentation](https://github.com/kubernetes/helm/blob/master/docs/rbac.md).
```
Comment (dunglas) [inline deployment/kubernetes.md]: Only this change should be keep. Other ones must be reverted: https://www.onlinegrammar.com.au/punctuation-in-lists/

**Comment by soyuka** [comment]
we already refused these changes: https://github.com/api-platform/docs/pull/942/files

**Pull request closed without merging**